extensions/scarb-doc/tests/core.rs

```diff
@@ -21,3 +23,25 @@ fn can_doc_corelib() {
         .assert()
         .success();
 }
+
+#[test]
+fn stdout_output_info() {
+    let t = TempDir::new().unwrap();
+    ProjectBuilder::start().name("hello_world").build(&t);
+
+    let metadata = Scarb::quick_snapbox()
+        .args(["--json", "metadata", "--format-version", "1"])
+        .current_dir(&t)
+        .stdout_json::<Metadata>();
+
+    Scarb::quick_snapbox()
+        .arg("doc")
+        .current_dir(&t)
+        .assert()
+        .success()
+        .stdout_matches(formatdoc! {r#"
+                Saved to directory: {dir}/doc
```

Comment:
Instead of 
```
Saved to directory: /private/var/folders/2t/7dt08_ss585ct08q57mvw_q40000gn/T/.tmpBguKBZ/target/doc
```

Please make it 

```
Saving output to: target/doc/hello_world/
```
And for `--output-format=json`
```
Saving output to: target/doc/output.json
```

You can take a look at scarb-execute/main.rs fn display_path for the prefix removal. I suggest "Saving" because we do "Compiling", "Running", etc for statuses - not sure if it's grammatically correct though 🤔 

Btw. `output.json` can be changed to `hello_world.json`, but that a separate problem :D

Reply:
Well, for sure it is grammatically correct, just might not exactly be the truth if the info is printed to the output after the files already exists. However, it makes perfect sense to change to "Saving" for the sake of consistency. Will do!